tests/TestCase/View/Helper/LayoutHelperTest.php

```diff
@@ -197,6 +197,75 @@ public function testModuleLink($expected, $name, array $viewVars = []): void
         static::assertSame($expected, $result);
     }
 
+    /**
+     * Data provider for `testTitle` test case.
+     *
+     * @return array
+     */
+    public function titleProvider(): array
+    {
+        return [
+            'empty' => [
+                '',
+                'Module',
+                [],
+            ],
+            'only module' => [
+                'objects',
+                'Module',
+                [
+                    'currentModule' => ['name' => 'objects'],
+                ],
+            ],
+            'objects' => [
+                'Object title | objects',
+                'Module',
+                [
+                    'object' => [
+                        'attributes' => [
+                            'title' => 'Object title',
+                        ],
+                    ],
+                    'currentModule' => ['name' => 'objects'],
+                ],
+            ],
+            'video' => [
+                'Video title | video',
+                'Module',
+                [
+                    'object' => [
+                        'attributes' => [
+                            'title' => 'Video title',
+                        ],
+                    ],
+                    'currentModule' => ['name' => 'video'],
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * Test `title` method
+     *
+     * @param string $expected The expected title
+     * @param string $name The view name
+     * @param array $viewVars The view vars
+     * @dataProvider titleProvider()
+     * @covers ::title()
+     */
+    public function testTitle($expected, $name, array $viewVars = []): void
```

Comment:
```suggestion
    public function testTitle(string $expected, string $name, array $viewVars = []): void
```
You can specify types in signature ;)